java/system-test/system-test-suite/src/test/java/sleeper/systemtest/suite/EmrPersistentBulkImportIT.java

```diff
@@ -49,6 +49,7 @@ public class EmrPersistentBulkImportIT {
     void setUp() throws InterruptedException {
         sleeper.connectToInstance(MAIN);
         sleeper.enableOptionalStack(PersistentEmrBulkImportStack.class);
+        sleeper.ingest().purgeQueue(BULK_IMPORT_PERSISTENT_EMR_JOB_QUEUE_URL);
```

Comment:
According to https://awscli.amazonaws.com/v2/documentation/api/latest/reference/sqs/purge-queue.html the purge queue operation can take up to 60 seconds. Do we need to wait or go back to the queue to check it's empty?

Reply:
Thanks, I've added `PurgeQueueDriver` and made it wait 60 seconds after the request is made